### PR TITLE
Welcome

### PR DESCRIPTION
**Pull request by denverdevelopments**
Neccesary checkmarks:

    [x] All Tests are Passing

    [x] The code will run locally

Type of change

    [x] New feature
    [] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

Testing Changes

    [] No Tests have been changed
    [x] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

    [] My code has no unused/commented out code
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def welcome
```
Comment (suzkiee) [inline app/controllers/users_controller.rb]: Do we need this action if it exists in the `SessionsController`?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      @user = User.new
+      # require 'pry';binding.pry
+  end
+
+  def create
+      user = user_params
+      user[:email] = user[:email].downcase
+      new_user = User.create(user)
+      flash[:success] = "Welcome, #{new_user.email}!"
+      redirect_to "/"
```
Comment (suzkiee) [inline app/controllers/users_controller.rb]: We should decide if we're using path helpers or not for consistency!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -6,15 +6,24 @@
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <style>
```
Comment (suzkiee) [inline app/views/layouts/application.html.erb:9]: This should live in `application.css`

File: app/views/sessions/welcome.html.erb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<h1 align="center">Welcome Viewing Party</h1>
+
+<section>
+  Viewing party is an app that allows you to search movies, and create viewing parties you host for friends.
+</section><br><br><br>
+
+<% if current_user %>
```
Comment (suzkiee) [inline app/views/sessions/welcome.html.erb:7]: Eyyyyyy this looks so good, Kevin!!!

File: spec/features/welcome_spec.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe 'logging in' do
+  it 'as a registered user, I can log in with correct credentials' do
+    user = User.create(email: 'test123@xyz.com', password: 'viewparty')
+
+    visit '/'
```
Comment (suzkiee) [inline spec/features/welcome_spec.rb:7]: Just flagging again for my path helpers question

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (denverdevelopments) [reply app/controllers/users_controller.rb]: You're right, it's already in SessionsCont., so not needed in UsersCont.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (denverdevelopments) [reply app/controllers/users_controller.rb]: I think the project desc. said to use path helpers 'where possible'.  So we can do it both ways, but I agree consistency would be nicer.  There was no path name for the root path, but we can setup an alias I guess.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (denverdevelopments) [reply app/views/layouts/application.html.erb:9]: Sounds fine to me, those lines are from the base Turing repo and was not added by us.

File: spec/features/welcome_spec.rb (hunk 5 above)
Comment (denverdevelopments) [reply spec/features/welcome_spec.rb:7]: See above response, thanks.

**Pull request merged**